api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -76,6 +76,9 @@ message ConnectionProperties {
 
   // Number of bytes sent to downstream.
   uint64 sent_bytes = 2;
+
+  // Details about why the connection was terminated.
+  string termination_details = 3;
```

Comment:
This should be emitted somewhere I would remove this for now. See my other comment though about stream vs connection.

Reply:
removed for now.